src/js/components/Modal/Modal.tsx

```diff
@@ -38,9 +40,17 @@ const Modal: React.FC<Props> = ({
     return () => {};
   }, []);
 
+  React.useEffect(() => {
+    document.body.classList.toggle('Modal--open', show);
+  }, [show]);
+
   return (
-    <div className={`Modal ${show ? 'Modal--open' : 'Modal--closed'}`}>
-      <div className="Modal-content u-posRelative" style={{ width: '50%', ...(style || {}) }}>
+    <div
```

Comment:
Let's add aria-label so that @jenshaeffer can target using Ghost Inspector. We can use something like this...

```JavaScript
   <div
      className={`Modal ${show ? 'Modal--open' : 'Modal--closed'} ${
        fullscreen ? 'Modal--fullscreen' : ''
      }`}
      aria-label="GI-modal"
    >
      <div
        className="Modal-content u-posRelative"
        style={{ ...(style || {}) }}
        aria-label="GI-modalContent"
      >
        <div
          className="Modal-header u-flex u-flexJustifyBetween"
          aria-label="GI-modalHeader"
        >
          <div className="u-sizeFill">
            <h2 aria-label="GI-modalHeaderText">{heading}</h2>
          </div>
          {showClose && (
            <div className="Modal-close">
              <Button
                type="link"
                onClick={() => closeFn?.()}
                icon="dismiss"
                mods="u-colorNeutral9"
                aria-label="GI-modalCloseButton"
              />
            </div>
          )}
        </div>
        <div className="Modal-body" aria-label="GI-modalBody">{children}</div>
      </div>
    </div>
```